src/annotator/config/index.js

```diff
@@ -8,7 +10,12 @@ import { toBoolean } from '../../shared/type-coercions';
  *
  * @typedef ConfigDefinition
  * @prop {(settings: SettingsGetters) => any} getValue -
- *   Method to retrieve the value from the incoming source
+ *  Method to retrieve the value from the incoming source
+ * @prop {boolean} allowInBrowserExt -
+ *  Allow this name to be available in the browser extension. If this is false
```

Comment:
```suggestion
 *  Allow this setting to be read in the browser extension. If this is false
```

I changed the terms here for clarity.